### PR TITLE
WRN-15493: Improve 'import skin' performance.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed the version of react to 18rc.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-15493

### Comments

### CODE REVIEW

File: sandstone/custom-skin/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		"@enact/ui": "^4.0.0",
 		"ilib": "^14.6.0",
 		"prop-types": "^15.6.2",
-		"react": "^17.0.1",
-		"react-dom": "^17.0.1"
+		"react": "^18.0.0-rc.0-next-fe905f152-20220107",
+		"react-dom": "^18.0.0-rc.0-next-fe905f152-20220107"
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/package.json:41]:
```suggestion
		"react": "^18.0.0-rc.0",
		"react-dom": "^18.0.0-rc.0"
```

**Review by daniel-stoian-lgp — APPROVED** [review]

File: sandstone/custom-skin/package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
 		"@enact/ui": "^4.0.0",
 		"ilib": "^14.6.0",
 		"prop-types": "^15.6.2",
-		"react": "^17.0.1",
-		"react-dom": "^17.0.1"
+        "react": "^17.0.1",
```
Comment (adrian-cocoara-lgp) [inline sandstone/custom-skin/package.json:40]: fix indentation

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: sandstone/custom-skin/package.json (hunk 2 above)
Comment (alexandrumorariu) [reply sandstone/custom-skin/package.json]: FIxed

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**